### PR TITLE
Adding Wordpress solution with Kubernetes

### PR DESCRIPTION
**Pull request by senpx**

### CODE REVIEW

File: _data/sidebars/home_sidebar.yml [hunk 1]
```diff
@@ -271,7 +271,11 @@
     - title: Docker Registry
       url: /applications/docker-registry.html
     - title: Wordpress
-      url: /applications/docker-compose.html
+      folderitems:
+      - title: Wordpress deployment by Docker
+        url: /applications/docker-compose.html
+      - title: Wordpress deployment by kubernetes
```
Comment (harsh-px) [inline _data/sidebars/home_sidebar.yml]: use capital k in kubernetes

**Comment by senpx** [comment]
Harsh, FYI, I have updated few more corrections. Can you please review and add your comment? I will update it accordingly. thanks in advance.

File: applications/wp_k8s.md [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+    volume.beta.kubernetes.io/storage-class: portworx-sc-repl3-shared
+spec:
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: 1Gi
+```
+
+## B.	 Create a Secret for MySQL Password
```
Comment (harsh-px) [inline applications/wp_k8s.md]: I just looked at the formatting and it's confusing. See screenshot below. Heading B is for secrets but the numbering continues for the sub-section after that. Also they need to be at different heading levels.

screen shot 2018-01-17 at 1 36 20 pm

Comment (senpx) [reply applications/wp_k8s.md]: Please let me know, If there anything need include or update it.

**Comment by harsh-px** [comment]
@senpx travis tests are failing

**Pull request merged**